VTEX - Logistics API.json

```diff
@@ -5003,433 +4929,307 @@
-                                        "PickupPointItemOptions",
-                                        "CanceledDateUtc",
-                                        "AuthorizedDateUtc",
-                                        "ConfirmedDateUtc",
-                                        "Errors",
-                                        "IsSucess"
-                                    ],
+                                "schema": {                                    
                                     "type": "object",
                                     "properties": {
                                         "LastUpdateDateUtc": {
-                                            "type": "string"
+                                            "type": "string",
+                                            "description": "Last update date and time in the format [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ`."
                                         },
                                         "SalesChannel": {
-                                            "type": "string"
+                                            "type": "string",
+                                            "description": "Sales channel ([trade policy](https://help.vtex.com/en/tutorial/how-trade-policies-work--6Xef8PZiFm40kg2STrMkMV)) valid for the reservation."
                                         },
                                         "LockId": {
-                                            "type": "string"
+                                            "type": "string",
+                                            "description": "Reservation ID."
                                         },
                                         "ReservationDateUtc": {
-                                            "type": "string"
+                                            "type": "string",
+                                            "description": "Reservation date and time in the format [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ`."
                                         },
                                         "MaximumConfirmationDateUtc": {
-                                            "type": "string"
+                                            "type": "string",
+                                            "description": "Maximum confirmation date and time in the format [ISO 8601 time zone offset format](https://learn.microsoft.com/en-us/rest/api/storageservices/formatting-datetime-values), as in `YYYY-MM-DDThh:mm:ss.ssZ`."
                                         },
                                         "Status": {
                                             "type": "integer",
-                                            "format": "int32"
```

Comment:
I suggest keeping this, it's useful to specify the format when possible, especially for people using tools like codegen

Reply:
Nice, did not know, I put it back and will keep them from now on, thanks!